src/utils/hact/work-order-complete/close-job.js

```diff
@@ -9,7 +9,7 @@ export const buildCloseWorkOrderData = (completionDate, notes, reference) => {
       {
         typeCode: 0,
         otherType: 'complete',
-        comments: notes,
+        comments: `Work order closed - ${notes}`,
```

Comment:
I noticed the format we use for cancellations is

`Cancelled: ${formData.cancelReason}`

Wonder if the user would get a little readability improvement if we keep them consistent? Just thinking about looking through a list of comments on a complex order. Maybe we could only conditionally show the `:` or `-` if there is a comment too..